esx_service/Makefile

```diff
@@ -89,25 +92,29 @@ build: $(VIB_BIN)
 TO_WDIR := $(DESCRIPTOR)
 TO_ESX_BIN := $(PY_SRC) $(PY_CLI) $(TOOLS)
 TO_ESX_PY  := $(PY_UTILS)
-TO_ESX_LIB := $(VMCI_SRV_LIB)
+TO_ESX_LIB := $(VMCI_SRV_LIBS)
 TO_ESX_INITD := $(INIT_SCRIPT)
 
 # copy files to staging area and then build the VIB
 $(VIB_BIN): $(TO_WDIR) $(TO_ESX_BIN) $(TO_ESX_PY) $(TO_ESX_LIB) $(TO_ESX_INITD)
 	@echo === Staging in $(PAYLOAD) ===
-	@mkdir -p $(VMDKOPS_BIN) $(VMDKOPS_PY) $(VMDKOPS_LIB) $(VMDKOPS_INITD)
+	@mkdir -p $(VMDKOPS_BIN) $(VMDKOPS_PY) $(VMDKOPS_LIB) $(VMDKOPS_LIB64) $(VMDKOPS_INITD)
 	@chmod -R a+w $(PAYLOAD)
 	@cp $(TO_WDIR) $(WDIR)
 	@cp $(TO_ESX_BIN) $(VMDKOPS_BIN)
 	@cp $(TO_ESX_PY)  $(VMDKOPS_PY)
-	@cp $(TO_ESX_LIB) $(VMDKOPS_LIB)
+	@cp $(VMCI_SRV_LIB) $(VMDKOPS_LIB)
+	@cp $(VMCI_SRV_LIB64) $(VMDKOPS_LIB64)/libvmci_srv.so
```

Comment:
please do not use the explicit name here. Define VMCI_SRV_LIB_NAME  as I suggested, or simply use $(basename)